types/luxon/index.d.ts

```diff
@@ -329,21 +334,21 @@ declare module 'luxon' {
                     seperator?: string;
                 }
             ): string;
-            toISO(options?: Object): string;
+            toISO(options?: ISOTimeOptions): string;
             toString(): string;
             union(other: Interval): Interval;
         }
 
-        type Settings = {
-            defaultLocale: string;
-            defaultNumberingSystem: string;
-            defaultOutputCalendar: string;
-            defaultZone: Zone;
-            defaultZoneName: string;
-            now: Function;
-            throwOnInvalid: boolean;
-            resetCache(): void;
-        };
+        class Settings {
+            static defaultLocale: string;
+            static defaultNumberingSystem: string;
+            static defaultOutputCalendar: string;
+            static defaultZone: Zone;
```

Comment:
Hah, I just saw this PR after I submitted one fixing the Settings. See #22494.

I defined it as a namespace to avoid the linting error preventing classes having only static methods, but I'm not sure this rule is a good one, especially since it forced me to define defaultZone as a `const`, although it's actually a read-only property.

This should be fixed here, as there is no setter for defaultZone in the JS code: see https://github.com/moment/luxon/blob/master/src/settings.js#L52

Reply:
I am thinking on this problem. How do you think about this?
```typescript
class Settings {
  static readonly defaultZone: Zone;
}
```
Yes, I'm not sure too. about disabling lint rules is not the good way or not.

Reply:
I think a class is closer to how the JS code is actually designed, and a readonly property is closer to the reality than a const variable. That's how I originally designed Settings in my PR, but the linter shouted at me, and I didn't want to take the responsibility of removing the linting rule, not being the original author of the typing. But I personally don't have a problem with removing it.

Reply:
I'll remove my PR, BTW, since this one solves the problem I wanted to solve.